<launch>

	<arg name="probability_threshold" default="0.8" />
	<arg name="count_th_" default="15" />

	<node name="traffic_sign_checker_node" pkg="traffic_sign_checker" type="traffic_sign_checker" output="screen">
		<param name="probability_threshold" value="$(arg probability_threshold)" />
		<param name="count_th" value="$(arg count_th_)"/>
	</node>

</launch>
